Sources/GeneralIOs/Services/FontService/FontService.swift

```diff
@@ -0,0 +1,299 @@
+//
+//  Created by Evgeny Schwarzkopf on 08.04.2022.
+//
+
+import Foundation
+import ArgumentParser
+import GeneralKit
+import AEXML
+import Stencil
+import PathKit
+
+public final class FontService {
+
+    private enum Constant {
+        static let notFoundTemplate = #"""
+                                      Is not found folder templates in directory.
+                                      Please, call command `general setup -r rosberry/general-templates\ ios`
+                                      """#
+    }
+
+    private enum Error: Swift.Error, LocalizedError {
+        case notFoundFonts(String)
+        case notFoundTarget(String)
+        case notFoundInfoPlist(String)
+        case invalidData
+        case notFoundTemplate
+        case somethingGoingWrong(String, String)
+
+        public var errorDescription: String? {
+            switch self {
+            case let .notFoundFonts(path):
+                return red("Is not found fonts in path \(path). Please check correctly path.")
+            case let .notFoundTarget(targetName):
+                return red("Is not found target - \(targetName)")
+            case let .notFoundInfoPlist(path):
+                return red("Is not found info plist by path \(path)")
+            case .invalidData:
+                return red("Data is not valid. Please check correctly Info.plist")
+            case .notFoundTemplate:
+                return red(Constant.notFoundTemplate)
+            case .somethingGoingWrong(let title, let subtitle):
+                return red("""
+                          Something going wrong...
+                          I try it \(title) ...
+                          But I can't perform because \(subtitle).
+                          Please check and try again.
+                          """)
```

Comment:
I think you should either place all text strings into constants or place them right here